userprofile/urls.py

```diff
@@ -12,5 +12,6 @@
         name="privacy_notice"),
     url(r'^accounts/$', views.ProfileView.as_view(),
         name="profile"),
+    url(r'^i18n/setlang', views.LanguageView.set_language),
     url(r'^i18n/', include('django.conf.urls.i18n')),
```

Comment:
I think you can drop this entyr, as it only adds the same url as above https://github.com/django/django/blob/b9cf764be62e77b4777b3a75ec256f6209a57671/django/conf/urls/i18n.py#L38

Reply:
I tried dropping it but the request seems to do nothing after that, because the request can't be connected to the right view i guess.

Reply:
Remember the `name` parameter as seen in the django source.

Reply:
Do you mean to drop the last url, because I don't understand how it works if I drop the `/setlang` url, like combine it with the `/accounts` url above?

This seems to work:
`url(r'^setlang', views.LanguageView.set_language,
     name="set_language")`

or this:
`url(r'^i18n/', views.LanguageView.set_language, name="set_language")`

Reply:
This `include('django.conf.urls.i18n')` shouldn't be needed at it only added `/accounts/i18n/setlang`, which you overwrote in the line above it, hence making the include line irrelevant (except it did add name for that url).

Reply:
Ok now I got it!